sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCScan.scala

```diff
@@ -40,7 +40,7 @@ case class JDBCScan(
   override def readSchema(): StructType = prunedSchema
 
   override def toV1TableScan[T <: BaseRelation with TableScan](context: SQLContext): T = {
-    new BaseRelation with TableScan {
+    new BaseRelation with TableScan with DescriptiveRelation {
```

Comment:
You can define a `class` with a descriptive name, cannot you?

Reply:
Yes, I was wondering what is better option, to define a class which extends BaseRelation or to define trait.
I ended up with trait because following reason.
If we define class whenever BaseRelation needs some new behaviour, we can possibly have a problem in the future when BaseRelation should for example needs to implement 2 interfaces (for example InsertableRelation and DescriptiveRelation). Another reason I wanted to use trait is that there are already traits like this in interfaces.scala (InsertableRelation, TableScan, PrunedScan, etc)

Reply:
I mean to replace the anonymous class `new BaseRelation with TableScan` by a private class out of the method:
```scala
private class JDBCRelation(context: SQLContext) extends BaseRelation with TableScan
```

Reply:
It is a little bit more flexible, since we can only log things than by class name, another reason is because there is already JDBCRelation, which does not extend v1 TableScan interface, so I have to name it on some different way or to maybe add TableScan trait to JDBCRelation (but I suppose there is some reason why that is not done yet)